qemu/tests/cfg/block_hotplug_with_max_map_count.cfg

```diff
@@ -0,0 +1,25 @@
+- block_hotplug_with_max_map_count:
+    only Linux
+    only virtio_blk
+    only x86_64
+    type = block_hotplug_with_max_map_count
+    virt_test_type = qemu
+    login_timeout = 240
+    start_vm = no
+    slots_mem = 20
+    maxmem_mem = 80G
+    mem = 4096
+    pcie_extra_root_port = 30
+    images += " stg1 stg2 stg3 stg4 stg5 stg6 stg7 stg8 stg9 stg10 stg11 stg12 stg13 stg14 stg15 stg16 stg17"
```

Comment:
suggest using extra_image_number to define the additional images, you already support dynamic create images. it is more flexible.

Reply:
Included !